xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -346,42 +301,40 @@ - (NSMenu*)applicationDockMenu:(NSApplication*)sender
 
 @end
 
-int main(int argc, char* argv[])
+int main(int argc, const char* argv[])
 {
+#if defined(DEBUG) && 0 // SHH enable as needed
```

Comment:
Any reason for disabling by default? Seems to be enabled on the old code if debug is set

Reply:
I don't think every debug build should enable this.  Do you know what it's for?

Reply:
No idea, but having different behaviour for release/debug is not great.

This seems to have been done by @Memphiz in https://github.com/xbmc/xbmc/commit/d6e57d210fbaa71243b0a375b71def6b0ccd45fe. Not sure if he's still around.